magento/magento2
# Fix: Add customer_address_id to billing_address

### Description (*)

`customer_address_id` will not set for new `billing_address`

### Fixed Issues (if relevant)

1. magento/magento2#17705: CustomerAddressId will not set for new billing adress

### Manual testing scenarios (*)


1. Create a new Customer Account
2. Add one Product to Cart
3. Create a New Shipment Address
4. Use Shipment Address as Order Address
5. Check `customer_address_id` with `address_type=billing` in `sales_order_address` 

### Questions or comments


### Contribution checklist (*)
